### PR TITLE
Offer Dashboard and First Cycle Page

### PR DESCRIPTION
**Pull request by bwormser**
Created Offer Dashboard with sample data to display functionality. Also, added create first cycle page for new users.

### CODE REVIEW

File: client/src/pages/FirstCycle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Box, Typography, useMediaQuery } from "@mui/material";
+import AddCircleOutlineOutlinedIcon from '@mui/icons-material/AddCircleOutlineOutlined';
```
Comment (GraceMattern) [inline client/src/pages/FirstCycle/index.tsx]: I would recommend using FAB component so it is an actual button and not an icon with a link

File: client/src/pages/FirstCycle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+        <>
+            <Box sx={{ display: 'flex', mt: 7, mb: 1, ml: 7 }}>
+                <div>
+                    <Typography sx={{ fontWeight: 'bold', fontSize: h1Size }} component="h1" variant="h4">Welcome to Next Round!</Typography>
+                    <Typography sx={{ display: 'flex', fontSize: bodySize, mt: 2}} variant="h2">Landing the job of your dreams is just a click away...</Typography>
+                </div>
+            </Box>
+            {/* TODO: Add onclick start cycle functionality */}
+            <AddCircleOutlineOutlinedIcon sx={{ color: 'blue', display: 'flex', fontSize: iconSize, margin: 'auto', mt: imagePad}}/>
```
Comment (GraceMattern) [inline client/src/pages/FirstCycle/index.tsx]: This should be grey (look at the grey on grey heart button [here](https://mui.com/material-ui/react-floating-action-button/)) or our accent color. This true blue is never used in our style guide

File: client/src/App/index.tsx [hunk 3]
```diff
@@ -23,6 +25,8 @@ const App = () => {
             <Route path={'/signin'} element={<SignIn />} />
             <Route path={'/signup'} element={<SignUp />} />
             <Route path={'/test_job'} element={<Job />} />
+            <Route path={'/offer_dashboard'} element={<OfferDash />} />
+            <Route path={'/first_cycle'} element={<FirstCycle />} />
```
Comment (GraceMattern) [inline client/src/App/index.tsx]: Minor suggestion but what if it was /offers and /create for the paths

**Review by GraceMattern — COMMENTED** [review]
Code looks fine overall but I think the create first cycle page could be nicer aesthetically

**Comment by mpolimen** [comment]
You should be able to select an offer on the offer dashboard and finalize your cycle by "accepting it". This should also present the option to post to social media

File: client/src/pages/OfferDash/index.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+import { useEffect, useState } from "react";
+import { Grid, Typography, useMediaQuery } from "@mui/material";
+import { Card, Avatar, CardHeader, CardContent } from "@mui/material";
+import { ApplicationObject, CycleObject} from "typings";
+
+export default function OfferDash () {
+    //(props: {user: ObjectId}) {
+
+    //Variable to populate cycle Info to match variables until api call is made
+    const tempCycle = {
```
Comment (mpolimen) [inline client/src/pages/OfferDash/index.tsx]: make sure to use your types!!!! this is a CycleObject

File: client/src/pages/OfferDash/index.tsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+        };
+
+        // TODO: Make api call to get application info
+        setOfferData(cycle["applications"]);
+    }, []);
+
+    return (
+        <>
+            <Typography sx={{ fontWeight: 'bold', fontSize: h1Size, ml: 7, mt: 7 }} component="h1" variant="h4">
+                Offer Dashboard:
```
Comment (mpolimen) [inline client/src/pages/OfferDash/index.tsx]: I say no semicolon

File: client/src/pages/OfferDash/index.tsx [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+            </Typography>
+            <Grid container sx={{ display: 'flex',  ml: 7, mt: 7 }}>
+                {offerData.map(offer => {
+                        return (
+                            <Grid sx={{ display: 'flex'}}>
```
Comment (mpolimen) [inline client/src/pages/OfferDash/index.tsx]: Since you are using map, each child needs a key. Look at the console output on the frontend

**Review by mpolimen — COMMENTED** [review]
Gave comments to Brian over text message. Needs to link fully with api hooks to be complete, and should add a modal for social media posting (like the form with post details). Also, needs an option to not post the offer, but finish anyway. Some minor visual stuff to adjust as well

**Comment by roccopolimen** [comment]
seems generally good to pull as long as you have tested that these work, we can merge and assess further changes if need be during our meeting.

**Pull request merged**